releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,16 @@
+---
+critical:
+  - |
+    In the newton and newton-rc branches openstack-ansible is referenced as a
+    submodule inside of the rpc-openstack path.  When a deployer is not using a
+    fresh clone of the rpc-openstack repository it is important to ensure that
+    the openstack-ansible SHA is pointing at the latest commit.  This can be
+    accomplished with the following commands run from the root of your
+    rpc-openstack path (i.e. /opt/rpc-openstack)::
+      $ git submodule update --init
+      $ git submodule sync
+      $ git submodule status
+
+    Ensure that the SHA output by the status command matches the HEAD of
```

Comment:
This will get out of date really quickly, so I'd suggest rather referring to a way to identify whether the rcbops submodule is correctly referenced. Perhaps just include the commands and example output in the shell text above?

Reply:
I added another section with the actual output, hopefully this makes it more clear.